test/units/cli/test_galaxy.py

```diff
@@ -30,13 +31,19 @@
 from ansible import context
 from ansible.cli.arguments import option_helpers as opt_help
 from ansible.cli.galaxy import GalaxyCLI
-from ansible.errors import AnsibleError, AnsibleOptionsError
-from ansible.module_utils.six import PY3
+from ansible.errors import AnsibleError
 from ansible.utils import context_objects as co
 from units.compat import unittest
 from units.compat.mock import call, patch
 
 
+@pytest.fixture(autouse='function')
+def reset_cli_args():
+    co.GlobalCLIArgs._Singleton__instance = None
+    yield
+    co.GlobalCLIArgs._Singleton__instance = None
```

Comment:
Maybe save the previous state and then restore it?

Reply:
I just copied it from another test, we run with `--boxed` that makes this largely irrelevant in CI but it helps when running it locally through a debugger.